agent/src/main/java/org/openremote/agent/protocol/tradfri/device/DeviceObserver.java

```diff
@@ -0,0 +1,96 @@
+package org.openremote.agent.protocol.tradfri.device;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.openremote.agent.protocol.tradfri.device.event.*;
+import org.openremote.agent.protocol.tradfri.payload.DeviceResponse;
+import org.openremote.agent.protocol.tradfri.util.CoapClient;
+
+import java.util.ArrayList;
+
+/**
+ * The class that observes a device to automagically detect changes
+ * @author Stijn Groenen
```

Comment:
We don't put author and version info in the javadocs...authors can be added to contributors.txt